glide.yaml

```diff
@@ -28,6 +28,10 @@ import:
   version: v2.6.0
   subpackages:
   - log
+- package: github.com/k8snetworkplumbingwg/network-attachment-definition-client
```

Comment:
Remove what, override with my fork? No we cannot because the official client lacks support for Tracker for fake clientset. As for why it doesn't have any open PRs, it's because I haven't posted any. And the reason for that is because we don't want to touch official clientset code that is generated by upstream code generator with the patch. Instead, we are waiting for https://github.com/kubernetes/kubernetes/pull/71049 to land, at which point we will retrigger client generator. Then we'll be able to get rid of the fork.